<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraSetting_Dialog</class>
 <widget class="QDialog" name="CameraSetting_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>相机参数设置界面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgsource.qrc">
    <normaloff>:/new/prefix1/images/photo-camera.svg</normaloff>:/new/prefix1/images/photo-camera.svg</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 63 12pt &quot;Bookman Old Style&quot;;</string>
   </property>
   <property name="text">
    <string>相机参数设置</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>241</width>
     <height>106</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>焦距(mm)：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="focalLength_lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Tx/Rx间距(mm)：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="camera_diff_lineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>积分次数：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="integrate_number_lineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="ok_pushButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="cancel_pushButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imgsource.qrc"/>
 </resources>
 <connections/>
</ui>
